README.md

```diff
@@ -1,64 +1,44 @@
 [![Build Status](https://travis-ci.org/tking2096/cs414-f17-301-chessire-coders.svg?branch=development)](https://travis-ci.org/tking2096/cs414-f17-301-chessire-coders)
 [![Stories in Ready](https://badge.waffle.io/tking2096/cs414-f17-301-chessire-coders.png?label=ready&title=Ready)](http://waffle.io/tking2096/cs414-f17-301-chessire-coders)
 
-# cs414-f17-301-chessire-coders
-Repository for the group project in CS414.
+## CSU CS414 - Chessire Coders
 
-## Online Board Game: Jungle
-A 1v1 strategy game that originated in China and resembles chess.
-[(wikipedia)](https://en.wikipedia.org/wiki/Jungle_(board_game))
+This is the project page for the ChesshireCoders, the best group in CSU's Fall 2017 CS414 class!
 
-## Installation & Setup
+[Developer Documentation](https://github.com/tking2096/cs414-f17-301-chessire-coders/wiki)
 
-Clone this repo to your local machine:
+## The Project
 
-`git clone https://github.com/tking2096/cs414-f17-301-chessire-coders.git`
+For this class, we were tasked with creating a Chess-based board game entirely in Java, and entirely from scratch. We were given the following requirments that our game system must fulfill:
 
-The entire project can be built with Maven using the following command:
+- There is a registration system in which new users can create an account in our game system.
+- User's must be able to un-register their accounts at any time.
+- An authentication system exists that allows registered users to log in given a correct username and password.
+- An invitation system must exist where any player can invite another player to participate in a game with them.
+- The reciever of the invitation has the choice to accept or reject the invitation.
+- A player may forfeit/quit a joined game at any time.
+- The game system must track each played game, including information about start and end times, as well as the outcome.
+- Each player has a player profile that lists the games he or she has played.
+- And of course, the game system must enforce the rules of the game.
 
-`mvn install`
+Our team has also chosen to implement a basic AI feature so that any player may play against a computer.
 
-Jar files will be built to each `<module>/target` directory.
+## Jungle
 
+Our team was tasked with creating an implementation of [Jungle](https://en.wikipedia.org/wiki/Jungle_(board_game)), a chess variant in which there are two players, each owning 8 pieces which have different power levels and special rules:
 
```

Comment:
You say power levels here but only give names below, do you think adding power level would be a good idea?

Reply:
I just wanted to give a brief overview of the game.